### ISSUE TITLE
Can't install this webui : cygheap base mismatch error

### ISSUE THREAD

**Issue by Djanghost** [vladmandic/automatic#285]
### Issue Description

Right after cloning the repository, when i run the webui.bat i've a bunch of this cygheap base mismatch error.
I installed all the prerequisites (python 10.6, cuda 12.1). I have A1111 webui too in another folder and it works well.
Already tried to reboot and search for cygwin1.dll to delete all but the most recent version but i only find one version. 
Hope someone could help :

![VLAD](https://user-images.githubusercontent.com/129005283/233393957-f4536529-f34d-4bd5-82c5-525be029ee8f.png)

![VLAD2](https://user-images.githubusercontent.com/129005283/233394384-4bc552ac-ac47-4f54-8af9-3d3f1afe5631.png)

### Version Platform Description

Version: e6701f5c Wed Apr 19 14:12:19 2023 -0400
OS : Windows 11
Browser : Chrome
GPU : RTX4090

**Comment by vladmandic** [comment]
i don't support cygwin installs on windows.  
there is no reason for cygwin nowadays when wsl2 is sooo much better in every aspect and built-in into windows.

**Comment by vladmandic** [comment]
And you started from wsl terminal, not cygwin?
It cannot be the same error as your first error is regarding cygheap and that has nothing to do with wsl

**Comment by Djanghost** [comment]
> And you started from wsl terminal, not cygwin? It cannot be the same error as your first error is regarding cygheap and that has nothing to do with wsl

if i understand i had to run webui.bat from wsl terminal ?
I can't do this or i don't know how to do, when i open wsl a wsl terminal, how can i run webui.bat with it?
Apparently i only can run webui.bat with powershell or cmd windows

**Comment by vladmandic** [comment]
if you use wsl, then you run webui.sh
if you use native windows, then you run webui.bat
either works.

**Comment by Djanghost** [comment]
> if you use wsl, then you run webui.sh if you use native windows, then you run webui.bat either works.

Thanks again for your time, i'm using native windows. But when i run webui.bat, cmd window open and all things seem running normaly until i get this sh*** cygwin mismatch errors. It's so frustrating :/
It's like if wsl was not detected or something ?

![vlad3](https://user-images.githubusercontent.com/129005283/233435342-2af666df-d875-4070-aa5b-a558e33c29bb.png)

**Comment by vladmandic** [comment]
yeah, sorry, its been over a decade since i've used cygwin. and this is clearly a cygwin error - even if you don't use it, it still sneaks in.

try searching on google or stackexchange for `cygheap base mismatch detected`.

**Comment by Djanghost** [comment]
> yeah, sorry, its been over a decade since i've used cygwin. and this is clearly a cygwin error - even if you don't use it, it still sneaks in.
> 
> try searching on google or stackexchange for `cygheap base mismatch detected`.

Finally solved the problem a few hours ago. All is working except Deforum extension. Deforum work when i did the first tests like 10 hours ago. I open a new issue. Congrats and full support for your work

**Comment by vladmandic** [comment]
Thanks.
There is already an open issue for Deforum and I'm working with extension author on fixes, don't create separate issue for that.

**Comment by Djanghost** [comment]
As you prefer but the issue you mention is about ControlNet. 
Well i just posted the issue i have in the same thread
